_monitoring-your-cluster/pa/reference.md

```diff
@@ -455,225 +531,498 @@ This list is extensive. We recommend using Ctrl/Cmd + F to find what you're look
-    <tr>
-      <td>Master_ThrottledPendingTasksCount (WIP)
-      </td>
-      <td>Sum of total pending tasks which got throttled by node (master node). It is a cumulative metric so look at the max aggregation.
-      </td>
-    </tr>
-    <tr>
-      <td>Election_Term (WIP)
-      </td>
-      <td rowspan="5">N/A
-      </td>
-      <td>Monotonically increasing number with every master election.
-      </td>
-    </tr>
-    <tr>
-      <td>PublishClusterState_Latency (WIP)
-      </td>
-      <td>The time taken by quorum of nodes to publish new cluster state. This metric is available for current master.
+      <td>The limit (in bytes) for how much memory operations can use.
       </td>
     </tr>
+   </tbody>
+  </table>
+
+## Relevant dimensions: `ClusterManagerTaskInsertOrder`, `ClusterManagerTaskPriority`, `ClusterManagerTaskType`, `ClusterManagerTaskMetadata`
+ 
+<table>
+ <thead style="text-align: left">
+   <tr>
+     <th>Metric</th>
+     <th>Description</th>
+   </tr>
+ </thead>
+ <tbody>  
+   <tr>
+     <td>ClusterManager_Task_Queue_Time
+     </td>
+     <td>The time (milliseconds) that a cluster manager task spent in the queue.
+     </td>
+   </tr>
+   <tr>
+      <td>ClusterManager_Task_Run_Time
+      </td>
+      <td>The time (milliseconds) that a cluster manager task has been executed.
+      </td>
+    </tr>
+ </tbody>
+</table>
+     
+## Relevant dimension: `CacheType` 
+  
+<table>
+  <thead style="text-align: left">
     <tr>
-      <td>PublishClusterState_Failure (WIP)
-      </td>
-      <td>The number of times publish new cluster state action failed on master node.
-      </td>
+     <th>Metric</th>
+     <th>Description</th>
+   </tr>
+  </thead>
+  <tbody>  
+   <tr>
+     <td>Cache_MaxSize
+     </td>
+     <td>The max size of the cache in bytes.
+     </td>
+   </tr>
+ </tbody>
+</table>
+
+## Relevant dimension: `ControllerName` 
+<table>
+ <thead style="text-align: left">
+  <tr>
+    <th>Metric</th>
+    <th>Description</th>
+  </tr>
+ </thead>
+ <tbody>  
+  <tr>
+    <td>AdmissionControl_RejectionCount 
+    </td>
+    <td>Total rejections performed by a Controller of Admission Control.
+    </td>
+  </tr>
+  <tr>
+    <td>AdmissionControl_CurrentValue 
+    </td>
+    <td>Current value for Controller of Admission Control.
+    </td>
+  </tr>
+  <tr>
+    <td>AdmissionControl_ThresholdValue
+    </td>
+    <td>Threshold value for Controller of Admission Control.
+    </td>
+  </tr>
+ </tbody>
+</table>
+
+## Relevant dimension: `NodeID` 
+  
+<table>
+  <thead style="text-align: left">
+   <tr>
+   <th>Metric</th>
+   <th>Description</th>
     </tr>
+  </thead>
+  <tbody>  
     <tr>
-      <td>ClusterApplierService_Latency (WIP)
+      <td>Data_RetryingPendingTasksCount 
       </td>
-      <td>The time taken by each node to apply cluster state sent by master.
+      <td>Number of throttled pending tasks on which the data node is actively performing retries. It is an absolute metric at that point of time.
```

Comment:
The second sentence here is a little awkward/unclear. Can we simplify/clarify? Something like "This is an absolute metric when..."